hil/cli/helper.py

```diff
@@ -10,24 +10,14 @@ def print_json(raw_output):
     sys.exit(0)
 
 
-def print_status_table(raw_output):
-    """Print the networking action status ID in a PrettyTable"""
-    status_table = PrettyTable()
-    status_table.field_names = ['Field', 'Value']
-    status_table.add_row(['Status ID', raw_output['status_id']])
-    print(status_table)
-    sys.exit(0)
-
-
-def print_list_table(raw_output, header):
-    """Prints a table with a single column. Also, puts the the counts
-    of items in the column name.
+def make_table(field_names=[], rows=[]):
```

Comment:
General rule of thumb: never use mutable objects as defaults for parameters. I don't think it's actually problematic in this case, but this sets off my spidey senses and I'd rather just write code that obviously doesn't have an issue.

FTR, the problem with using a list as the default arises in cases like this:


```python
def append_and_print(lst=[]):
    lst.append(1)
    print(lst)

>>> append_and_print()
[1]
>>> append_and_print()
[1, 1]
>>> append_and_print()
[1, 1, 1]
>>> append_and_print()
[1, 1, 1, 1]
...
```

The value is evaluated at the time of definition, so if you make modifications to it, the next call to the function uses the modified version...

I think an empty tuple would work in this case.

Reply:
> The value is evaluated at the time of definition, so if you make modifications to it, the next call to the function uses the modified version...

oh! thanks for letting me know about this. 

> I think an empty tuple would work in this case.

I don't understand. Where would I use an empty tuple? I can just not use defaults in this case and it will work, right?
```python
def make_table(filed_names, rows):
```